### DISCUSSION TITLE
VRCX slow issue

### DISCUSSION THREAD

**Discussion by Rex234** [vrcx-team/VRCX#274] [General]
Anyone else experiencing slow responses from VRCX? Like it takes ages for me to click on anything.. maybe its loading too much shit for me. I have 1K friends but I don't think it's an issue right?

Im not sure what I can do to make vrcx faster for me.

**Comment by Myrkie** [comment]
Haven't noticed any issues could be API related maybe a few of my friends have been having some slow API response times recently for both VRCX and Vrchat.net

**Reply by Rex234** [reply]
Its just very laggy for me. I think its just how much shit is being logged in my feed. cuz of how many ppl i have changing avatars and locations etc etc.

If i log into an account with lesser friends, Its smooth

**Reply by Myrkie** [reply]
if thats the case you could be getting rate limited by vrchat for a lot of requests vrcx is inherently unoptimized according to PYPY himself.

**Reply by Rex234** [reply]
Just meant the app itself is very slow and laggy. 

Anyways, filtering my feed to like "status" or something that is less logged on the feed seems to make it smoother for me. thanks

**Reply by Natsumi-sama** [reply]
Press `Ctrl + Shift +I` and in the console run `$app.debugWebRequests = true;` you will be able to see all of the web requests being made.

**Reply by Natsumi-sama** [reply]
What's your total feed size?

**Reply by Myrkie** [reply]
> Press `Ctrl + Shift +I` and in the console run `$app.debugWebRequests = true;` you will be able to see all of the web requests being made.

was completely unaware of this thankies.

**Reply by Rex234** [reply]
> What's your total feed size?

My feed size? it's usually around 30 - 60k. I always try to clear it often

**Reply by Myrkie** [reply]
60k? jesus mines not even 7k

**Reply by Rex234** [reply]
> 60k? jesus mines not even 7k

It just depends. When I started this discussion i cleared it
so an hour later.. rn... its at 34k

**Reply by Natsumi-sama** [reply]
oh wow yea that's way too big I will need to make a limit for that

**Reply by Myrkie** [reply]
> oh wow yea that's way too big I will need to make a limit for that

15k maybe? 34k in under an hour jeez

**Reply by Rex234** [reply]
> oh wow yea that's way too big I will need to make a limit for that

oof. thank you. I have like 263 ppl online right now my feed always gets filled quickly haha. appreciate the replies

**Reply by Myrkie** [reply]
> > oh wow yea that's way too big I will need to make a limit for that
> 
> oof. thank you. I have like 263 ppl online right now my feed always gets filled quickly haha. appreciate the replies

I only got 30 people online no wonder its chugging badly for you q.q

**Reply by Rex234** [reply]
> > > oh wow yea that's way too big I will need to make a limit for that
> > 
> > 
> > oof. thank you. I have like 263 ppl online right now my feed always gets filled quickly haha. appreciate the replies
> 
> I only got 30 people online no wonder its chugging badly for you q.q

Its sign for me to stop adding randoms or ppl on the spot

**Reply by SalbugVR** [reply]
Yeah, mine does chug a bit as well especially when a lot of people are online and my feed starts to fill up quickly.

**Reply by Rex234** [reply]
> Yeah, mine does chug a bit as well especially when a lot of people are online and my feed starts to fill up quickly.

glad im not the only one

**Comment by Rex234** [comment]
Hey I just want to put out a follow up and say ty for applying that 5000 log limiter.

VRCX hasn't been slowing down, chugging, anymore! Apps been smooth since.

Ty for your work and appreciate.
